public/src/components/channelManagement/bannerDeploy/bannerDeployChannelDeployer.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from 'react';
 
-import { Button, Theme, withStyles, createStyles, WithStyles } from '@material-ui/core';
```

Comment:
the use of `WithStyles` caused an annoying bug that prevented me from using styles inside a nested component, so I've replaced it with `useStyles` hooks for all bannerDeploy components